.env.example

```diff
@@ -91,13 +91,21 @@ export BOOKWAREHOUSE_NAMESPACE=bookwarehouse
 # export ENABLE_EGRESS=true
 
 # optional: ENABLE_GRAFANA (true/false)
-# Default: false
+# Default: true
```

Comment:
Isn't the default false ? OSM by default won't deploy Prometheus or Grafana

Reply:
These defaults are for the demo script, not the osm cli itself